### ISSUE TITLE
Qhull imported targets hard to consume

### ISSUE THREAD

**Issue by SpaceIm** [qhull/qhull#70]
Currently, Qhull creates different imported targets for its components depending on shared and debug.
It makes it hard to properly consume Qhull in another library. Indeed, downstream build system shouldn't care weither Qhull was built as Debug or static/shared (it should be final app decision).

It would be nice to be able to consume Qhull like this:

```cmake
find_package(Qhull REQUIRED libqhull libqhull_r CONFIG)
target_link_libraries(otherlibA PRIVATE Qhull::libqhull)
target_link_libraries(otherlibB PRIVATE Qhull::libqhull_r)
```

instead of something tedious like this:
```cmake
option(QHULL_SHARED "" OFF)
find_package(Qhull REQUIRED)
if(QHULL_SHARED)
  if(CMAKE_BUILD_TYPE MATCHES "[dD]ebug")
    target_link_libraries(otherlibA PRIVATE Qhull::qhull_d)
    target_link_libraries(otherlibB PRIVATE Qhull::qhull_rd)
  else()
    target_link_libraries(otherlibA PRIVATE Qhull::libqhull)
    target_link_libraries(otherlibB PRIVATE Qhull::qhull_r)
  endif()
else()
  if(CMAKE_BUILD_TYPE MATCHES "[dD]ebug")
    target_link_libraries(otherlibA PRIVATE Qhull::qhullstatic_d)
    target_link_libraries(otherlibB PRIVATE Qhull::qhullstatic_rd)
  else()
    target_link_libraries(otherlibA PRIVATE Qhull::qhullstatic)
    target_link_libraries(otherlibB PRIVATE Qhull::qhullstatic_r)
  endif()
endif()
```

**Comment by SpaceIm** [comment]
Even worse, for multi generators, debug targets have the name of non debug targets.

**Comment by SpaceIm** [comment]
If you don't mind, I'll submit a PR to give the same name to CMake imported target and pkgconf files for Debug and non Debug build type (static/shared will still have different name to not break too much existing bevahiour), while preserving libs file names (add _d or d suffix if Debug, using generator expression).

**Comment by cbbarber** [comment]
Fixed via push request #76
